### ISSUE TITLE
Add stock nodes - pulled from FlyBase

### ISSUE THREAD

**Issue by dosumis** [VirtualFlyBrain/VFB_neo4j#31]
```SQL
SELECT f.uniquename, s.uniquename FROM feature f 
JOIN feature_genotype fg ON f.feature_id = fg.feature_id 
JOIN stock_genotype sg on fg.genotype_id = sg.genotype_id 
JOIN stock s on s.stock_id=sg.stock_id limit 3;
```

 | uniquename  | uniquename  |
|-------------|------------- |
| FBal0000021 | FBst0000002 |
| FBal0005583 | FBst0000002 |
| FBba0000025 | FBst0000002 |

Stock IDs can be used for linkouts to FlyBase via the usual recipe.

**Comment by jcolomb** [comment]
s.description will work only for BDSC flies.
you need to use the flybase name to be consistent over different stock centers, but then you have to work the name which comes on the form of a list of: @flybasealleleid:fballelename@ (note that it is using insertion name and not allele name for insertions, note that unknown allele comes in simple text (w[¨*];) without any "@") 
you may check Rflybase

**Comment by dosumis** [comment]
You can use the feature_relation table to look up insertion /transposon names from alleles.  This should => you a closer match to names used in stock centers.

**Comment by jcolomb** [comment]
Yep this should work. I would give better names for fly stock list in the lab. Thanks for the tip.

By the way, both flybase and stock centers use cryptic names for RNAi lines...

Dr. Julien Colomb
#Neurobiology #openscience
http://orcid.org/0000-0002-3127-5520

Schillerpromenade 4
12049 Berlin
0049 30 46721825

**Comment by dosumis** [comment]
The feature_relation table also => links to genes.  This would, presumably, help for RNA lines.

**Comment by dosumis** [comment]
(RNA -> RNAi)
